Src/GeneratedTests/Discovery.Tests/resources/urlshortener.json

```diff
@@ -0,0 +1,319 @@
+﻿{
```

Comment:
Do we need all of this to prove that it works? I understand the benefits of using the real resource, but if we cut off all the parameters etc, would we actually lose anythign?